src/foundry/foundry.js/hooks.d.ts

```diff
@@ -341,53 +346,51 @@ declare global {
       ) => boolean | void;
 
       /**
-       * @remarks This is called before the {@link Game} is initialized for the current window location.
+       * A hook event that fires as Foundry is initializing, right before any initialization tasks have begun.
        * @remarks This is called by {@link Hooks.callAll}.
        * @see {@link Game#initialize}
        */
       init: () => unknown;
 
       /**
-       * @remarks This is called when initializing shaders for a {@link PointSource}.
-       * @param pointSource   - the PointSource to initialize shaders for
-       * @param animationType - a key used in `CONFIG.Canvas.lightAnimations`
+       * A hook event that fires after PointSource shaders have initialized.
+       * @param source        - The PointSource
```

Comment:
The parameters in the function definition should be renamed according to the documentation.